extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/ChangeRecordImpl.java

```diff
@@ -22,92 +22,78 @@
 import com.hazelcast.jet.cdc.RecordPart;
 
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.Serializable;
 import java.util.Map;
-import java.util.Objects;
 
-public class ChangeRecordImpl implements ChangeRecord {
+import static java.util.Objects.requireNonNull;
+
+public class ChangeRecordImpl implements ChangeRecord, Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
We implement `SerializerHook` for the `ChangeRecordImpl` so the Serializable shouldn't be needed.